main.js

```diff
@@ -3,7 +3,8 @@
  */
 
 import * as key from './lib/key/cli.js';
+import * as id from './lib/id/cli.js';
 
 export {
```

Comment:
I'd prefer these to be named exports:

```js
export key;
export id;
```

Bundling them up into an object is an anti-pattern / holdover from CommonJS days. Even if it doesn't matter much as an internal file in a CLI module, people may copy this elsewhere so we should avoid it.

Reply:
@dlongley got it, thanks.